cypress/e2e/podcast.ts

```diff
@@ -0,0 +1,27 @@
+describe("podcast tests", () => {
+  it("Navigate to podcasts home page", () => {
+    cy.visit("http://localhost:3000/")
+
+    // podcasts page will be the third link in nav bar
+    cy.get("nav").find("a").contains("Podcast").click()
+    cy.url().should("eq", "http://localhost:3000/podcast")
+  })
+
+  it("Check Title", () => {
+    cy.visit("http://localhost:3000/podcast")
+    cy.get('#Title').contains('Trash Turtle Football')
```

Comment:
Nit: You generally want to use a data-testid attribute instead of a regular id attribute to target elements in tests, because this way it signifies that it's for the test and not styling, as well as refactoring purposes. Here's a link about it as well: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements. I always have used data-testid since it gets used in RTL as well, but any of those are fine, really.